samples/ios/app/glean-sample-app.xcodeproj/project.pbxproj

```diff
@@ -498,9 +495,9 @@
 					"$(inherited)",
 					"@executable_path/Frameworks",
 				);
-				PRODUCT_BUNDLE_IDENTIFIER = "rediger.net.glean-sample-app";
+				PRODUCT_BUNDLE_IDENTIFIER = "net.tlong.glean-sample-app";
```

Comment:
🤔 - we should really put that into some mozilla namespace. followup?

Reply:
Already thought of that :D https://bugzilla.mozilla.org/show_bug.cgi?id=1582840